english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/getonbus/getonbus.md

```diff
@@ -2,6 +2,4 @@ You and the KitKat-shaped warrior board the old and rusted bus.
 The driver looks at you both suspiciously and asks you where you would like to go...
 You stare at your new friend and both of you know what's next. You yell triumphantly that you're headed for...
 
-[The Flaming Tower](journey-to-flaming-tower/journey-to-flaming-tower.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.